modules/deployment-plug-in-cluster.adoc

```diff
@@ -39,3 +39,8 @@ This command runs a light-weight NGINX HTTP server to serve the assets for your
 ----
 $ oc patch consoles.operator.openshift.io cluster --patch '{ "spec": { "plugins": ["my-plugin"] } }' --type=merge
 ----
+
+[NOTE]
+====
+If you do not see your plug-in, you might need to refresh your browser if the plugin was recently enabled. If you recieve any errors at runtime, check the JS console in browser dev tools any errors in your plugin code.
```

Comment:
Worth mentioning that it can take a few minutes for the new plugin configuration to roll out. Also there is a UI where you can see the list of the enabled plugins (either Home -> Overview or Cluster Settings -> Global configuration -> Console -> Console plugins)